src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/index.tsx

```diff
@@ -100,15 +100,16 @@ const ContractInteraction: React.FC<ContractInteractionProps> = ({
       >
         {(submitting, validating, rest, mutators) => {
           setCallResults = mutators.setCallResults
-
           return (
             <>
               <Block className={classes.formContainer}>
                 <SafeInfo />
                 <FormDivisor />
                 <EthAddressInput
+                  value={rest.values.contractAddress}
```

Comment:
You can get any form's field value, by using the `useField` hook, inside `EthAddressInput`

```typescript
const {
  input: { value: contractAddress }
} = useField('contractAddress', { subscription: { value: true } })
```